### DISCUSSION TITLE
Para aquellos que aún no se deciden por el lenguaje de Back...

### DISCUSSION THREAD

**Discussion by carlos-gmx3** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#579] [General]
Les comparto algunos links
[Java y JavaScript ¿son lo mismo? ](https://www.campusmvp.es/recursos/post/Java-y-JavaScript-son-lo-mismo.aspx)
[¿Conoces la diferencía en Java y JavaScript?](https://www.softzone.es/programas/lenguajes/diferencias-java-javascript/)

**Comment by DanyVeneno** [comment]
Gracias Carlos

**Comment by jstoledano** [comment]
![image](https://user-images.githubusercontent.com/2502353/161405946-4d3145ce-5c90-4fc0-84d4-7a841f92f105.png)

**Comment by SergioPinto98** [comment]
Alguien tiene la liga de la reunion que hubo para escoger?
